src/Dapr.Workflow/WorkflowServiceCollectionExtensions.cs

```diff
@@ -57,7 +59,19 @@ public static class WorkflowServiceCollectionExtensions
 
                 if (TryGetGrpcAddress(out string address))
                 {
-                    builder.UseGrpc(address);
+                    string? apiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
+                    if (!string.IsNullOrEmpty(apiToken))
+                    {
+                        serviceCollection.ConfigureDurableGrpcClient(apiToken);
+                        HttpClient client = new HttpClient();
+                        client.DefaultRequestHeaders.Add("Dapr-Api-Token", apiToken);
+                        builder.UseGrpc(CreateChannel(client));
```

Comment:
This seems to repeat the code in the `ConfigureDurableGrpcClient` do we need both methods?